hibernate-core/src/test/java/org/hibernate/test/joinedsubclassbatch/Person.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+//$Id: Person.java 10218 2006-08-04 18:24:04Z steve.ebersole@jboss.com $
+package org.hibernate.test.joinedsubclassbatch;
+
+
+import org.hibernate.annotations.AccessType;
+import org.hibernate.annotations.AttributeAccessor;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+
+/**
+ * @author dcebotarenco
+ */
+public class Person {
+	@Id
```

Comment:
Why did you choose to use HBM since you already use annotations?

Reply:
copy/paste